package.json

```diff
@@ -21,46 +21,46 @@
     "url": "https://github.com/taskcluster/aws-provisioner.git"
   },
   "dependencies": {
-    "assume": "^1.2.5",
-    "aws-sdk": "^2.23.0",
-    "azure-blob-storage": "^1.0.2",
-    "azure-entities": "^1.2.1",
-    "azure-storage": "^1.0.1",
+    "assume": "^1.5.0",
+    "aws-sdk": "^2.41.0",
+    "azure-blob-storage-temp-fork": "^1.0.4",
```

Comment:
Not that there is any reason to change it now... Just a general recommendation... Pollution of npm is not a nice thing to do...